src/Email/Body.php

```diff
@@ -25,9 +25,9 @@ class Body
     protected $html;
 
     /**
-     * @var string|null
+     * @var string
      */
-    protected $text;
+    protected $text = '';
```

Comment:
@lukasbestle @distantnative  I think it makes more sense to make the default values of the protected attributes in the Body and Email classes inside the attribute methods, instead of making them empty strings. What do you think?

Like that:

````php
protected $text;

public function text()
{
    return $this->text ?? '';
}
````

Reply:
Maybe the easier fix indeed.

Reply:
I mean like that https://github.com/getkirby/kirby/commit/59ad5a6d2c7297e2e6de729818ca48038ca68cab . Which way should we choose?

Reply:
I’ll leave that completely up to you as it’s the choice of how Kirby wants to deal with this. My suggestion is just using PHPMailer’s code method but if you’re using the one @afbora mentioned everywhere else already it may make sense to use this here, too.

Reply:
Actually the problem is: the solution in this PR may not work. Because setters can overwrite `null` on property. That's why I looked for a different solution.

Reply:
Thanks for explaining it further. What’d be the case that it wouldn’t work? My current code change solves my error and works in my case but of course I’m not sure if it works in all cases. If your solution is the more robust one though I tend to choose it but leaving ultimately to the Kirby team.

Reply:
As you said, it may be working now, but in edge cases, the same problem may occur again.

Reply:
Please check out the PR I'm proposing https://github.com/getkirby/kirby/pull/4170